template/sym-lib-table

```diff
@@ -0,0 +1,85 @@
+(sym_lib_table
+  (lib (name 74xgxx)(type Legacy)(uri ${KICAD_SYS_SYMBOL_DIR}/74xgxx.lib)(options "")(descr "Legacy 74xgxx symbol library."))
```

Comment:
Why not keep the current format of KiCad variable name: `KISYSSYM` and `KIPRJSYM` ?